modules/cluster-management/src/integrationTest/java/org/apache/ignite/internal/cluster/management/rest/RestTestBase.java

```diff
@@ -40,6 +41,7 @@
  * Cluster management REST test.
  */
 @MicronautTest
+@ExtendWith(ConfigurationExtension.class)
```

Comment:
Why do you need this extension here? This class doesn't use configuration anywhere

Reply:
removed